runbooks/KubevirtHyperconvergedClusterOperatorCRModification.md

```diff
@@ -14,15 +14,15 @@ When the operands are changed by someone else constantly, it may lead to oscilla
 
 ## Diagnosis
 
-Check the alert details. `component_name` is the name of the operand that is overwritten.
+Check the alert details. `component_name` refers the operand that is overwritten and it follows `<kind-of-operand>/<name-of-operand>` pattern.
```

Comment:
@erkanerol  Hi, I believe it should be "refers to"

Reply:
Fixed.